### PR TITLE
Better explanation of contribution guidelines

### PR DESCRIPTION
**Pull request by KeizerDev**
Maybe add to the README file to first read this.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,6 +4,11 @@
...
 
-1. When adding a `provider` use for each a new **single** commit.
-2. Use gofmt on all files.
-3. Refactor commits to the base code should be discussed in an issue prior to PR. 
\ No newline at end of file
+1. When adding a `provider`: 
+    1. Use for each a new **single** commit.
+    2. Add it to the README.md
+    3. Add it to `s.go`
+    4. Keep it all alphabetically
```
Comment (zquestz) [inline CONTRIBUTING.md]: alphabetical.

Comment (KeizerDev) [inline CONTRIBUTING.md]: alphabetical sorted?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,6 +4,11 @@
...
+    5. Create a PR.
+2. When doing refactor changes to the base code, first this should be discussed in an issue prior to PR.
```
Comment (zquestz) [inline CONTRIBUTING.md]: "When doing refactors to the base code, make sure to discuss it in an issue prior to PR."

File: CONTRIBUTING.md (hunk 1 above)
Comment (zquestz) [inline CONTRIBUTING.md]: "Make sure providers in README.md and s.go are alphabetically sorted."

**Comment by KeizerDev** [comment]
Fixed

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -4,6 +4,11 @@
 
 There are a few basic ground-rules for contributors to `s`:
 
-1. When adding a `provider` use for each a new **single** commit.
-2. Use gofmt on all files.
-3. Refactor commits to the base code should be discussed in an issue prior to PR. 
\ No newline at end of file
+1. When adding a `provider`: 
+    1. Use for each a new **single** commit.
```
Comment (zquestz) [inline CONTRIBUTING.md]: Each provider should be a **single** commit.

**Comment by zquestz** [comment]
Merge-able after the last NIT is fixed.

**Comment by KeizerDev** [comment]
Done :smiley:

**Comment by zquestz** [comment]
Money.

**Comment by KeizerDev** [comment]
Lol

**Comment by KeizerDev** [comment]
So? It can be merged?

**Comment by zquestz** [comment]
I would say yes. Or you can add one more line.

```
4. Profit!
```

=)

**Comment by KeizerDev** [comment]
Lol, finally then :wink:

**Pull request merged**